spec/view_helper/description_spec.rb

```diff
@@ -47,9 +47,21 @@
     end
   end
 
-  it 'should truncate correctly' do
+  it 'should truncate at 200 chars, by default' do
```

Comment:
Prefer to retain single quotes as that is the general style in the rest of the gem.